### ISSUE TITLE
8va and 8vb lines request

### ISSUE THREAD

**Issue by musikboden** [paulrosen/abcjs#408]
It would be nice to have the !8va(! !8va)! working
![Skärmavbild 2020-05-26 kl  12 39 12](https://user-images.githubusercontent.com/18282700/82891383-f0d4d680-9f4d-11ea-95dc-7f5c631ce946.png)

**Comment by paulrosen** [comment]
I agree - I don't know when I'll get to it, but if I'm in the neighborhood I'll work on it.

**Comment by musikboden** [comment]
> I agree - I don't know when I'll get to it, but if I'm in the neighborhood I'll work on it.

Great!
